source/pda_aq_programmer.c

```diff
@@ -638,7 +657,7 @@ void *set_aqualink_PDA_device_on_off( void *ptr )
 
   LOG(PDA_LOG,LOG_INFO, "PDA Device On/Off, device '%s', state %d\n",aq_data->aqbuttons[device].label,state);
 
-  if (! goto_pda_menu(aq_data, PM_EQUIPTMENT_CONTROL)) {
+  if (! goto_pda_menu(aq_data, PM_EQUIPTMENT_CONTROL, true)) {
```

Comment:
This could waste time when you turn off multiple devices because you are already on the equipment control menu.